packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenuItemWrapper/ContextualMenuSplitButton.tsx

```diff
@@ -117,12 +117,15 @@ export class ContextualMenuSplitButton extends ContextualMenuItemWrapper {
       // tslint:disable:deprecation
       name: item.name,
       text: item.text || item.name,
+      secondaryText: item.secondaryText,
       // tslint:enable:deprecation
       className: classNames.splitPrimary,
       canCheck: item.canCheck,
       isChecked: item.isChecked,
       checked: item.checked,
       iconProps: item.iconProps,
+      onRenderIcon: item.onRenderIcon,
```

Comment:
Ah so #12196 is an issue because the lack of onRenderIcon in itemProps is preventing your custom function from being called?

Reply:
yes, that's what it is.